README.md

```diff
@@ -1 +1,119 @@
-# Leaderboard-
\ No newline at end of file
+# Leaderboard
+
+<a name="readme-top"></a>
+
+<div align="center">
+  <h3><b>Leaderboard
+</b></h3>
+</div>
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Usage](#usage)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+
+# 📖 Mobile First Project <a name="about-project"></a>
+
+In this project I created a nice to-do-list app to track my daily tasks
+
+## 🛠 Built With <a name="built-with"></a>
+
+1. HTML
+2. JavaScript
+
+### Key Features <a name="key-features"></a>
+
+- leaderBoard website displays scores submitted by different players. It also allows you to submit your score. All data is preserved
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- [Live Demo Link](https://majdalkilany.github.io/To-Do-List/dist/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+1. A Web browser.
+2. A Code editor
+3. A Version cotrol sytem(Gt-msc)
+
+### Setup
+
+Clone this repository to your desired folder:
+
+Run these command:
+
+`https://majdalkilany.github.io/To-Do-List/dist/`
+
+### Usage
+
+To run the project, do the following:
+
+Open index.html using a live server.
+
+## 👥 Authors <a name="authors"></a>
+
+👤 Majd Alkilany
+
+- GitHub: [@majdalkilany](https://github.com/majdalkilany)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🔭 Future Features <a name="future-features"></a>
+
+- [ ] Add a navigation System
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](https://github.com/SiveMdluli/Leaderboard/issues).
```

Comment:
- [ ]  kindly consider adding the right path to the issue page of this repo to make your readme more interactive and professional